### DISCUSSION TITLE
v1.8.0 release

### DISCUSSION THREAD

**Discussion by catcherwong** [dotnetcore/EasyCaching#441] [Announcements]
## What's Changed
* Added a HybridCachingOption to throw an error if the distributed cache throws an error by @JudeVajira in https://github.com/dotnetcore/EasyCaching/pull/413
* feat(memory): support evicted event hook(#294) by @catcherwong in https://github.com/dotnetcore/EasyCaching/pull/414
* fix: typo ‘BaseGetDatabse’ by @InCerryGit in https://github.com/dotnetcore/EasyCaching/pull/417
* feat(FasterKv):  Add FasterKv support by @InCerryGit in https://github.com/dotnetcore/EasyCaching/pull/418
* fix: use DefaultObjectPoolProvider to get a DisposableObjectPool inst… by @sarmis in https://github.com/dotnetcore/EasyCaching/pull/427
* feat: restore queue model and consumer on disconnection by @sarmis in https://github.com/dotnetcore/EasyCaching/pull/428
* Add GetAllKeysByPrefix to InMemory and Redis by @meysamhadeli in https://github.com/dotnetcore/EasyCaching/pull/422
* chore(deps): bump Newtonsoft.Json from 12.0.3 to 13.0.2 in /src/EasyCaching.SQLite by @dependabot in https://github.com/dotnetcore/EasyCaching/pull/431
* chore(deps): bump Newtonsoft.Json from 12.0.3 to 13.0.2 in /serialization/EasyCaching.Serialization.Json by @dependabot in https://github.com/dotnetcore/EasyCaching/pull/432
* zremrangebyscore support for both Redis and CSRedis by @amiru3f in https://github.com/dotnetcore/EasyCaching/pull/436
* MemoryPack Serializer by @amiru3f in https://github.com/dotnetcore/EasyCaching/pull/438

## New Contributors
* @InCerryGit made their first contribution in https://github.com/dotnetcore/EasyCaching/pull/417
* @meysamhadeli made their first contribution in https://github.com/dotnetcore/EasyCaching/pull/422
* @dependabot made their first contribution in https://github.com/dotnetcore/EasyCaching/pull/431
* @amiru3f made their first contribution in https://github.com/dotnetcore/EasyCaching/pull/436

**Full Changelog**: https://github.com/dotnetcore/EasyCaching/compare/v1.7.0...v1.8.0

<hr /><em>This discussion was created from the release <a href='https://github.com/dotnetcore/EasyCaching/releases/tag/v1.8.0'>v1.8.0 release</a>.</em>

**Comment by amiru3f** [comment]
@catcherwong When will the EasyCaching.Serialization.MemoryPack NuGet package be released? We need to use in our env

**Reply by catcherwong** [reply]
https://www.nuget.org/packages/EasyCaching.Serialization.MemoryPack

Released!
